src/project/projecttypes.h

```diff
@@ -141,6 +141,10 @@ struct ProjectMeta
     size_t partsCount = 0;
     QPixmap thumbnail;
     QDate creationDate;
+    QString movementNumber;
```

Comment:
Now we do not support movements, so these fields are not needed

Reply:
This is __not__ about albums, on the contrary, it is for separate scores, one score per movement, for a larger piece sharing the same "main" title.